lib/segment/src/index/field_index/numeric_index/mutable_numeric_index.rs

```diff
@@ -1,28 +1,57 @@
 use std::collections::BTreeSet;
 use std::ops::Bound;
 use std::ops::Bound::{Excluded, Unbounded};
+use std::path::PathBuf;
 use std::sync::Arc;
 
 use common::counter::hardware_counter::HardwareCounterCell;
 use common::types::PointOffsetType;
+use gridstore::config::StorageOptions;
+use gridstore::{Blob, Gridstore};
 use parking_lot::RwLock;
 use rocksdb::DB;
 
 use super::mmap_numeric_index::MmapNumericIndex;
 use super::{
     Encodable, HISTOGRAM_MAX_BUCKET_SIZE, HISTOGRAM_PRECISION, numeric_index_storage_cf_name,
 };
+use crate::common::Flusher;
 use crate::common::operation_error::{OperationError, OperationResult};
 use crate::common::rocksdb_buffered_delete_wrapper::DatabaseColumnScheduledDeleteWrapper;
 use crate::common::rocksdb_wrapper::DatabaseColumnWrapper;
 use crate::index::field_index::histogram::{Histogram, Numericable, Point};
 use crate::index::field_index::mmap_point_to_values::MmapValue;
 
-pub struct MutableNumericIndex<T: Encodable + Numericable> {
-    db_wrapper: DatabaseColumnScheduledDeleteWrapper,
+/// Default options for Gridstore storage
+const fn default_gridstore_options<T: Sized>() -> StorageOptions {
+    let block_size = size_of::<T>();
+    StorageOptions {
+        // Size of numeric values in index
+        block_size_bytes: Some(block_size),
+        // Compressing numeric values is unreasonable
+        compression: Some(gridstore::config::Compression::None),
+        page_size_bytes: Some(block_size * 8192 * 32), // 4 to 8 MiB = block_size * region_blocks * regions,
```

Comment:
Why having pages of 4 to 8MB instead of default of 32MB used for payload storage & sparse vectors?

Reply:
Because the block size is smaller, the idea was to keep the same number of blocks in a page, rather than to put more blocks in a page and keep the same size. If I understand it correctly we now have less regions per page, this way we prevent long searches through gaps.

To be honest, these values are still a bit arbitrary. The idea is to 'tune' them once Gridstore has been implemented for all payload indices. There's still a lot of things changing, so if we tune it now, we might not get the best result in the end. Would you be okay with that?

Reply:
> For each index we create, we allocate at least one page on disk. If you have an empty mutable segment, it still creates this first page. If you have many payload indices the problem becomes more noticeable. 
In this case we just scaled down the page size together with the block size.

Thanks this makes sense!
Can you please add a comment to capture this decision please :pray: